app/Http/Controllers/API/EmployeeVendorController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\MemberInviteRequest;
+use App\Models\UserRole;
+use App\Models\VendorEmployee;
+use App\Services\AdminService;
+use App\Services\InviteService;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Mockery\Exception;
+
+class EmployeeVendorController extends Controller
+{
+    public function deleteUser(Request $request){
+        $employeeVendor=VendorEmployee::where('id',$request->id)->first();
+        if (!$employeeVendor) {
+            return response()->json([
+                'message' => 'User does not exist with given id'
+            ],Response::HTTP_NOT_FOUND);
+        }
+        AdminService::deleteRoles($request->id);
```

Comment:
@Babita3331 You are accessing the models directly in the controller and you are using service classes. Please do some research on the general practice of using service layers.